vipps-order-management-api.md

```diff
@@ -66,11 +66,11 @@ in the Getting started guide, for details.
 
 ### OrderId and PaymentType
 
-The idea with order management is to add a Receipt or Category to a Vipps transaction made with the Ecom or Recurring API. So, the receipt needs to be connected to a `OrderId`. The `OrderId` is what **you** use when either initiating a Ecom payment or creating a recurring charge.
+The idea with order management is to add a Receipt or Category to a Vipps transaction made with the eCom or Recurring API. So, the receipt needs to be connected to a `OrderId`. The `OrderId` is what **you** use when either initiating an eCom payment or creating a recurring charge.
```

Comment:
I think yes, it is probably time to revise the whole file to refer to ePayment. I think it's more than replacing the name. The ePayment API uses "reference" instead of "OrderId", I think, and there will be links and things. You will know best. I think that should be done by you guys though.

Reply:
👍